src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [inputData, setFilteredList] = useState('');
+  const filteredList = moviesFromServer.filter(
+    movie => movie.title.toLowerCase()
+      .includes(inputData.toLocaleLowerCase().trim())
+      || movie.description.toLowerCase()
+        .includes(inputData.toLocaleLowerCase().trim()),
+  );
```

Comment:
The handleChanges function can be named more descriptively, like handleInputChange or handleSearchInputChange.